pandas/tests/scalar/period/test_period.py

```diff
@@ -1030,7 +1038,8 @@ def test_sub_delta(self):
         result = left - right
         assert result == 4 * right.freq
 
-        with pytest.raises(IncompatibleFrequency):
+        msg = "Input has different freq=M from Period\\(freq=A-DEC\\)"
```

Comment:
Nitpick, not a blocker.

```suggestion
        msg = r"Input has different freq=M from Period\(freq=A-DEC\)"
```